pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchSink.java

```diff
@@ -195,7 +214,7 @@ public Pair<String, String> extractIdAndDocument(Record<GenericObject> record) t
 
             String id = null;
             if (!elasticSearchConfig.isKeyIgnore() && key != null) {
-                if (keySchema != null){
+                if (keySchema != null) {
```

Comment:
It is not recommended to include format changes in a PR.

Reply:
I updated code to reduce diff as possible